src/views/Ifos/components/IfoQuestions/index.tsx

```diff
@@ -43,7 +43,7 @@ const IfoQuestions = () => {
             </Heading>
           </CardHeader>
           <CardBody>
-            {config.map(({ title, description }, i, { length }) => (
+            {config(t).map(({ title, description }, i, { length }) => (
```

Comment:
@memoyil shouldn't you remove the `t(...)` down here? Since it comes here already translated from your updates.

Reply:
Done